src/MLDatasets.jl

```diff
@@ -150,6 +152,7 @@ function __init__()
     __init__fashionmnist()
     __init__mnist()
     __init__svhn2()
+    __init__omniglot()
```

Comment:
also here alphavetic ordering

Reply:
Changed. I noticed that some sections don't have alphabetical ordering. Would it be worth opening a new pull request with these consistency fixes?

Reply:
yes, any help in cleaning up the docs or the code is well accepted